x-pack/test/security_solution_cypress/cypress/e2e/exceptions/shared_exception_lists_management/manage_exceptions.cy.ts

```diff
@@ -46,13 +46,12 @@ describe(
   { tags: ['@ess', '@serverless', '@skipInServerless'] },
   () => {
     beforeEach(() => {
-      cy.task('esArchiverResetKibana');
+      login();
```

Comment:
Interesting that `login()` isn't required to be done before ES data manipulation helpers as `rootRequest()` helper under the hood uses `ELASTICSEARCH_USERNAME` and `ELASTICSEARCH_PASSWORD` for auth.